src/commands/get/status.js

```diff
@@ -47,10 +70,14 @@ class StatusCommand extends Command {
     const { pkg, version } = this.parsePackage(args.package);
     const route = flags.events ? 'status-events' : 'status';
     this.config = this.mergeConfig(flags);
-    this.config.host = 'https://' + this.config.host;
+    const { statusHost: host } = this.config;
+
+    if (!host) {
+      return console.log('Missing warehouse status host. Please configure `~/.wrhs` config file, or use the `--status-host` option.');
```

Comment:
Maybe output a link to the github readme showing what the file should look like?  Also for people updating their wrhs and getting this change they will see this error even-though they have a `~/.wrhs` config file already, it just won't have the status host in it.